types/braintree-web/modules/apple-pay.d.ts

```diff
@@ -130,6 +130,46 @@ export class ApplePaySession {
     onshippingmethodselected: (event: any) => void;
 
     onvalidatemerchant: (event: any) => void;
+
+    /**
+     * The requested action succeeded.
+     */
+    static readonly STATUS_SUCCESS: number;
```

Comment:
Q: What is the difference between these static entries and the ones defined up in `ApplePayStatusCodes`? When would you use one vs the other?

Reply:
These fields are in the right place now, according to apple documentation. Also you can see that in Braintree documentation they use it as ApplePaySession.STATUS_SUCCESS...
But I left ApplePayStatusCodes enum for back compatibility so no-one's code wouldn't break if remove this enum

Reply:
Gotcha, thank you! Could add an `/** @deprecated */` comment to the `ApplePayStatusCodes` enum and point them to these so people stop using them? PR looks good otherwise!

Reply:
yep, I just added it :+1:

Reply:
Hi @jbuckner, can you please approve the merge?)